styles/venture/layout.s2

```diff
@@ -1217,118 +1228,138 @@ $module_layout_css
 .one-column #content .module-content { text-align: left;}
 
 .two-columns-left #content,
-.three-columns-sides #content { border-left: $*sidebar_width solid transparent; }
+.three-columns-sides #content { border: none; float: none; }
 
-.two-columns-right #content,
-.three-columns-sides #content { border-right: $*sidebar_width solid transparent; }
+.column-right #content { min-width: $*sidebar_width; box-sizing: border-box; }
 
-.three-columns-left #content { border-left: $*sidebar_width_doubled solid transparent; }
+#content { padding: 1em; }
```

Comment:
This is messing with the margins being set on line 1201 - you may need to change that one to just left and right and pull it into a media query (so desktop sizes get the margins but they collapse down on mobile)

Reply:
oh good point... okay, i think i've unfucked this